lhotse/recipes/libritts.py

```diff
@@ -148,8 +148,8 @@ def prepare_libritts(
             #   84_121123_000008_000000 Villefort rose, half ashamed of being surprised in such a paroxysm of grief.    Villefort rose, half ashamed of being surprised in such a paroxysm of grief.
 
             # book.tsv contains additional metadata
-            utt2snr = {
-                rec_id: float(snr)
+            utt2snr = [
```

Comment:
I don't understand why this change is needed (you can access utterance ids later with utt2snr.keys())

Reply:
I need them sorted as they appear in the file. The order is not kept during initialisation via kwargs arguments what this syntax actually boils down to. Plus Python 3.6 does not support OrderedDict behaviours by default on standard dict anyways only Python 3.7+ does.

Reply:
Python 3.6 does support that actually. I think they made it a guaranteed behavior in 3.7 but the change was made one release earlier. Anyway we can keep these changes.